extract_unique_sensors_01.py

```diff
@@ -0,0 +1,39 @@
+from urllib.request import urlopen
+
+url = ('http://monctl.devm.camlab.kat.ac.za/kat/doc/manuals/sensors/SensorList.txt')
+try:
+    sensors_data = urlopen(url)
+except Exception:
+    filename = ('/home/lebukani/Downloads/SensorList.txt')
+    with open(filename) as lines:
+        sensors_data = lines.readlines()
+print(sensors_data)
```

Comment:
This can be refactored to a function that takes in a URL and file path, and returns the sensor data.